### PR TITLE
Feature/catalog

### PR DESCRIPTION
**Pull request by kamarabbas99**
hero section without the catalog api

### CODE REVIEW

File: package-lock.json
Comment (KunalRajpal) [inline package-lock.json:1]: dev: true is used to run npm run dev

File: src/app/(pages)/[slug]/page.tsx [hunk 1]
```diff
@@ -52,11 +57,23 @@ export default async function Page({ params: { slug = 'home' } }) {
...
-      <Hero {...hero} />
-      <Blocks
-        blocks={layout}
-        disableTopPadding={!hero || hero?.type === 'none' || hero?.type === 'lowImpact'}
-      />
+      {slug === 'home' ? (
+        <section>
+          <Hero {...hero} />
+          <Gutter>
+            {/* Add rest of the stuff */}
```
Comment (KunalRajpal) [inline src/app/(pages)/[slug]/page.tsx:64]: what is supposed to go here? "{/* Add rest of the stuff */}"

File: src/app/_heros/CustomHeroCatalog/index.module.scss [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+  position: relative;
+}
+
+
+
+/* Container for the form */
+.formContainer {
+  text-align: left;
+  background-color: transparent;
+  // background-color: #f3f3f3; /* Light grey background */
```
Comment (KunalRajpal) [inline src/app/_heros/CustomHeroCatalog/index.module.scss:66]: if this is to be removed, remove the comment

File: src/app/_heros/CustomHeroCatalog/index.module.scss [hunk 3]
```diff
@@ -0,0 +1,107 @@
...
+  padding: 10px;
+  border-radius: 5px;
+  border: 1px solid #ccc;
+}
+
+/* Style for the search button */
+.searchButton {
+  width: 100%;
+  padding: 10px;
+  background-color: #007BFF; /* Bootstrap blue */
```
Comment (KunalRajpal) [inline src/app/_heros/CustomHeroCatalog/index.module.scss:91]: can make a variable for this if reused

File: src/app/_heros/CustomHeroCatalog/index.tsx [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+    event.preventDefault();
+    console.log({ year, make, model, engine });
+  };
+  
+  const mediaUrl = 
+   media &&
+   typeof media !== 'string' && `${process.env.NEXT_PUBLIC_SERVER_URL}/media/${media.filename}`
+  return (
+    <section className={classes.hero}>
+      <div className={classes.heroWrapper} style={{  backgroundImage: `linear-gradient(rgba(0, 0, 0, 0.7), rgba(0, 0, 0, 0.7)), url(${mediaUrl})`,  backgroundSize: 'cover', backgroundPosition: 'center', backgroundRepeat: 'no-repeat'}}>
```
Comment (KunalRajpal) [inline src/app/_heros/CustomHeroCatalog/index.tsx:31]: make a variable for the gradient color

File: src/app/_heros/CustomHeroCatalog/index.tsx [hunk 5]
```diff
@@ -0,0 +1,80 @@
...
+
+        <div className={classes.heroTextBox}>
+          <RichText content={richText} />
+          <div className={classes.formContainer}>
+            <form onSubmit={handleSubmit}>
+              <div className={classes.selectField}>
+                <Label>Year
+                  <select value={year} onChange={(e) => setYear(e.target.value)}>
+                    <option value="2022">2022</option>
+                    {/* Add more years as needed */}
```
Comment (KunalRajpal) [inline src/app/_heros/CustomHeroCatalog/index.tsx:41]: remove comment

File: src/app/_heros/CustomHeroCatalog/index.tsx [hunk 6]
```diff
@@ -0,0 +1,80 @@
...
+                  </select>
+                </Label>
+              </div>
+              <div className={classes.selectField}>
+                <Label>Make
+                  <select value={make} onChange={(e) => setMake(e.target.value)}>
+                    <option value="Subaru">Subaru</option>
+                    {/* Add more makes as needed */}
```
Comment (KunalRajpal) [inline src/app/_heros/CustomHeroCatalog/index.tsx:49]: remove comment

File: src/app/_heros/CustomHeroCatalog/index.tsx [hunk 7]
```diff
@@ -0,0 +1,80 @@
...
+                  </select>
+                </Label>
+              </div>
+              <div className={classes.selectField}>
+                <Label>Model
+                  <select value={model} onChange={(e) => setModel(e.target.value)}>
+                    <option value="Legacy">Legacy</option>
+                    {/* Add more models as needed */}
```
Comment (KunalRajpal) [inline src/app/_heros/CustomHeroCatalog/index.tsx:57]: remove comment

**Review by KunalRajpal — APPROVED** [review]

**Review by RayanImran — APPROVED** [review]

**Review by hassannauman123 — APPROVED** [review]

**Pull request merged**